healthcheck/ticker.go

```diff
@@ -55,7 +55,7 @@ func (ticker *ticker) runCheck(ctx context.Context, wg *sync.WaitGroup) {
 		if ticker.check.state != nil {
 			name = ticker.check.state.Name()
 		}
-		log.Event(nil, "failed", log.Error(err), log.Data{"external_service": name})
+		log.Error(context.Background(), "failed", err, log.Data{"external_service": name})
```

Comment:
use ctx value instead of creating new context
```suggestion
		log.Error(ctx, "failed", err, log.Data{"external_service": name})
```

Reply:
@wavemechanics @thomasridd - context.Background() wasn't the right sub for nil

Reply:
If there is a ctx variable set, always use `ctx` otherwise instantiating ctx value with context.Background() is okay.

We tend to always pass ctx through function signatures as it contains information that we want to display in the logs - e.g. trace id

Reply:
understood - fixed